rclpy/src/rclpy/_rclpy_handle.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Python.h>
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
+
+#include "rclpy_common/handle.h"
+
+size_t
```

Comment:
Why not `void *`?

Reply:
Not `void *` because I [don't see a built in conversion for it](https://pybind11.readthedocs.io/en/stable/advanced/cast/overview.html#list-of-all-builtin-conversions). I added a `static_assert()` that `size_t` is big enough for the job in 1910a61